package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/wallet-sdk",
-  "version": "3.0.4",
+  "version": "3.0.5",
```

versions aren't bumped in PRs
```suggestion
  "version": "3.0.4",
```